crates/consensus/consensus/src/lib.rs

```diff
@@ -450,6 +450,9 @@ pub enum ConsensusError {
         /// The block's timestamp.
         timestamp: u64,
     },
+    /// Other, likely an L2 error.
+    #[display("other, search logs for l2 error")]
+    Other,
```

Comment:
this variant is temp, is possible to remove since https://github.com/paradigmxyz/reth/pull/13655 lays way for this, but grows scope of this pr way too large since is hardcoded `Error = ConsensusError` for use `Consensus` trait in many places in codebase

Reply:
let me see if we can relax the clone here then we can use Other(Box here